carriers/ups.js

```diff
@@ -0,0 +1,216 @@
+                    state: activity.ActivityLocation.Address === undefined ? (activity.ActivityLocation.StateProvinceCode === undefined ? '' : activity.ActivityLocation.StateProvinceCode) : (activity.ActivityLocation.Address.StateProvinceCode === undefined ? '' : activity.ActivityLocation.Address.StateProvinceCode),
+                    country: activity.ActivityLocation.Address === undefined ? (activity.ActivityLocation.CountryCode === undefined ? '' : activity.ActivityLocation.CountryCode) : (activity.ActivityLocation.Address.CountryCode === undefined ? '' : activity.ActivityLocation.Address.CountryCode),
+                    zipcode: activity.ActivityLocation.Address === undefined ? (activity.ActivityLocation.PostalCode === undefined ? '' : activity.ActivityLocation.PostalCode) : (activity.ActivityLocation.Address.PostalCode === undefined ? '' : activity.ActivityLocation.Address.PostalCode)
+                }
+                activity.location = geography.addressToString(activity.address);
+            } else {
+                activity.address = {
+                    city: '',
+                    state: '',
+                    country: '',
+                    zipcode: ''
+                }
+
+                activity.location = '';
+            }
+        })
+    } else {
+        activitiesList.address = {
+            city: activitiesList.ActivityLocation.Address === undefined ? (activitiesList.ActivityLocation.City === undefined ? '' : activitiesList.ActivityLocation.City) : (activitiesList.ActivityLocation.Address.City === undefined ? '' : activitiesList.ActivityLocation.Address.City),
+            state: activitiesList.ActivityLocation.Address === undefined ? (activitiesList.ActivityLocation.StateProvinceCode === undefined ? '' : activitiesList.ActivityLocation.StateProvinceCode) : (activitiesList.ActivityLocation.Address.StateProvinceCode === undefined ? '' : activitiesList.ActivityLocation.Address.StateProvinceCode),
+            country: activitiesList.ActivityLocation.Address === undefined ? (activitiesList.ActivityLocation.CountryCode === undefined ? '' : activitiesList.ActivityLocation.CountryCode) : (activitiesList.ActivityLocation.Address.CountryCode === undefined ? '' : activitiesList.ActivityLocation.Address.CountryCode),
+            zipcode: activitiesList.ActivityLocation.Address === undefined ? (activitiesList.ActivityLocation.PostalCode === undefined ? '' : activitiesList.ActivityLocation.PostalCode) : (activitiesList.ActivityLocation.Address.PostalCode === undefined ? '' : activitiesList.ActivityLocation.Address.PostalCode)
+        }
+
+        activitiesList.location = geography.addressToString(activitiesList.address);
+    }
+
+    return activitiesList;
+}
+
+function filter(res) {
+    const packageInfo = res.body.TrackResponse.Shipment.Package;
+    var activitiesList = [];
+    if(packageInfo === undefined){
```

Comment:
`if(` should be `if (` (needs a space)
I'm wondering if the ESLint rules for this project are missing/incorrect. There are a few lines (like this one) that should be throwing ESLint errors per our current style.
@freshlogic space-after-if and no-space-after-function seem to be missing (I made those names up), right?